push_combined_summary_to_es.py

```diff
@@ -46,6 +51,13 @@
                 # remove timestamps from author's names and collect unique names only
                 xml_file_data['authors'] = remove_timestamps_from_author_names(xml_file_data['authors'])
 
+            updated_at = xml_file_data['updated']
+
+            if (now - updated_at) > timedelta(days=2):
+                continue
+
+            del xml_file_data['updated']
```

Comment:
As we need to push only a newly generated or recently updated combined summary in the ES index (avoiding the overwriting behavior), we are parsing this field `updated` from the XML file to check for the recent update in the particular file. 

If the XML file has been updated in the past 2 days we push it to the ES index, else we skip it. Here we are also remove the `updated` field (that we parsed earlier while reading the XML file) - because this `updated` field is not required in the ES index and the ES index schema stays as it is.

Here we have added 2 days as a fallback strategy so that if due to any reason if a cron job fails for a day - The combined summary file gets pushed on the next day.